자릿수더하기.js

```diff
@@ -0,0 +1,19 @@
+function solution(n){
+    if (n > 100000000 ){
+        return n;
+    }
+    let trans = (n + '').split('');
+    console.log(n)
+    if (n == 0){
```

Comment:
```suggestion
    if (n === 0) {
```
==은 자료형까지 비교하지 않습니다 ex) 3 == '3' => true
===을 사용해서 확실하게 비교해주세요